pydcs_extensions/SWPack/SWPack.py

```diff
@@ -205,137 +157,104 @@ class YWINGA(PlaneType):
         )
-        AGM_88C_HARM___High_Speed_Anti_Radiation_Missile = (
+        Kh_29T__AS_14_Kedge____670kg__ASM__TV_Guided__ = (
             3,
-            Weapons.AGM_88C_HARM___High_Speed_Anti_Radiation_Missile,
+            Weapons.Kh_29T__AS_14_Kedge____670kg__ASM__TV_Guided__,
         )
+        Mk_84___2000lb_GP_Bomb_LD = (3, Weapons.Mk_84___2000lb_GP_Bomb_LD)
 
     class Pylon4:
-        LAU_88_with_3_x_AGM_65D___Maverick_D__IIR_ASM_ = (
-            4,
-            Weapons.LAU_88_with_3_x_AGM_65D___Maverick_D__IIR_ASM_,
-        )
-        PROTONMissile = (
+        PROTONB = (4, Weapons.PROTONB)
+        Kh_58U__AS_11_Kilter____640kg__ARM__IN__Pas_Rdr = (
             4,
-            SWWeapons.PROTONMissile,
+            Weapons.Kh_58U__AS_11_Kilter____640kg__ARM__IN__Pas_Rdr,
         )
-        Mk_84___2000lb_GP_Bomb_LD = (
+        Kh_29T__AS_14_Kedge____670kg__ASM__TV_Guided__ = (
             4,
-            Weapons.Mk_84___2000lb_GP_Bomb_LD,
-        )
-        AGM_88C_HARM___High_Speed_Anti_Radiation_Missile = (
-            4,
-            Weapons.AGM_88C_HARM___High_Speed_Anti_Radiation_Missile,
+            Weapons.Kh_29T__AS_14_Kedge____670kg__ASM__TV_Guided__,
         )
+        Mk_84___2000lb_GP_Bomb_LD = (4, Weapons.Mk_84___2000lb_GP_Bomb_LD)
 
     class Pylon5:
-        LAU_88_with_3_x_AGM_65D___Maverick_D__IIR_ASM_ = (
-            5,
-            Weapons.LAU_88_with_3_x_AGM_65D___Maverick_D__IIR_ASM_,
-        )
-        PROTONMissile = (
-            5,
-            SWWeapons.PROTONMissile,
-        )
-        Mk_84___2000lb_GP_Bomb_LD = (
+        Kh_58U__AS_11_Kilter____640kg__ARM__IN__Pas_Rdr = (
             5,
-            Weapons.Mk_84___2000lb_GP_Bomb_LD,
+            Weapons.Kh_58U__AS_11_Kilter____640kg__ARM__IN__Pas_Rdr,
         )
-        AGM_88C_HARM___High_Speed_Anti_Radiation_Missile = (
+        Kh_29T__AS_14_Kedge____670kg__ASM__TV_Guided__ = (
             5,
-            Weapons.AGM_88C_HARM___High_Speed_Anti_Radiation_Missile,
+            Weapons.Kh_29T__AS_14_Kedge____670kg__ASM__TV_Guided__,
         )
+        Mk_84___2000lb_GP_Bomb_LD = (5, Weapons.Mk_84___2000lb_GP_Bomb_LD)
 
     class Pylon6:
-        PROTONMissile = (6, SWWeapons.PROTONMissile)
-        LAU_88_with_3_x_AGM_65D___Maverick_D__IIR_ASM_ = (
-            6,
-            Weapons.LAU_88_with_3_x_AGM_65D___Maverick_D__IIR_ASM_,
-        )
-        Mk_84___2000lb_GP_Bomb_LD = (
-            6,
-            Weapons.Mk_84___2000lb_GP_Bomb_LD,
-        )
-        AGM_88C_HARM___High_Speed_Anti_Radiation_Missile = (
-            6,
-            Weapons.AGM_88C_HARM___High_Speed_Anti_Radiation_Missile,
-        )
+        Mk_84___2000lb_GP_Bomb_LD = (6, Weapons.Mk_84___2000lb_GP_Bomb_LD)
+        L_081_Fantasmagoria_ELINT_pod = (6, Weapons.L_081_Fantasmagoria_ELINT_pod)
 
     class Pylon7:
-        ENERGY_CELL___ = (7, SWWeapons.ENERGY_CELL___)
+        ENERGY_CELL___ = (7, Weapons.ENERGY_CELL___)
```

Comment:
Custom ordnance like this needs to refer to SWWeapons.ENERGY_CELL___
Otherwise it will cause exceptions, I presume as soon as you open up the payload tab in Retribution

Reply:
Fixed